### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by FrancisBehnen**
Correct properties file name and add mailcatcher command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,13 @@ IntelliJ has some neat functionality to make running the application very easy.
 #### Set the properties
 The API has a number of properties that need to be set before the applicaiton can run.
 There's a template in `resources/config/application.properties.sample`.
-Copy the contents from this file into a `application-dev.properties`.
+Copy the contents from this file into a `application.properties`.
 This file is gitignored because it contains private information such as possible database credentials and API keys.
 The sample proprties assume a working PostgreSQL installation running in the background.
 
+#### Mailcatcher
+Before you can start the server you should start mailcatcher. This should work by just launching `mailcatcher` from your terminal.
```
Comment (TimvdLippe) [inline README.md]: Might also want to add how to install mailcatcher?

**Comment by martijnjanssen** [comment]
Good fix, thanks!

Next time, please try to adhere to the branch naming conventions as stated in the readme.

File: README.md (hunk 1 above)
Comment (martijnjanssen) [inline README.md]: I have an entire new readme on my machine, I just haven't had the time to include it. It is included in that one. I could open a PR with the new updated readme.

**Comment by FrancisBehnen** [comment]
Are you referring to the missing prefix or the underscore? I did not think this deserved the fix prefix, and you said to me yesterday that we use underscores between words in branch names (contrary to what is in the readme).

**Comment by martijnjanssen** [comment]
I was referring to the prefix, the branch name is completely fine with me :)

File: README.md (hunk 1 above)
Comment (TimvdLippe) [inline README.md]: Ping :wink:

Comment (martijnjanssen) [inline README.md]: Oh damn, adding it to my todo-list!

**Comment by martijnjanssen** [comment]
Closing this PR after opening #319.

**Pull request closed without merging**